core/components/minishop2/lexicon/uk/setting.inc.php

```diff
@@ -77,6 +77,8 @@
 
 $_lang['setting_ms2_cart_handler_class'] = 'Клас обробника кошика';
 $_lang['setting_ms2_cart_handler_class_desc'] = 'Імʼя класу, що реалізує логіку роботи з кошиком.';
+$_lang['setting_ms2_cart_context'] = 'Використовувати єдину кошик для всіх контекстів?';
+$_lang['setting_ms2_cart_context_desc'] = 'Якщо включено, то використовується загальний кошик для всіх контекстів. Якщо вимкнено - то у кожного контексту використовується своя кошик.';

```

Comment:
```suggestion
$_lang['setting_ms2_cart_context_desc'] = 'Якщо включено, то використовується загальний кошик для всіх контекстів. Якщо вимкнено - то у кожного контексту використовується своя кошик.';

```
$_lang['setting_ms2_cart_context_desc'] = 'Якщо включено, то використовується загальний кошик для всіх контекстів. Якщо вимкнено - то у кожного контексту використовується свій кошик.';